Tests/PostgreSQLTests/PostgreSQLTests.swift

```diff
@@ -103,14 +103,15 @@ extension Album: ModelProtocol {
 // MARK: - Tests
 
 public class PostgreSQLTests: XCTestCase {
-    let connection = try! PostgreSQL.Connection(info: .init(URL(string: "postgres://localhost:5432/swift_test")!))
+    var connection: Connection!
 
     let logger = Logger(name: "SQL Logger", appenders: [StandardOutputAppender()])
 
     override public func setUp() {
         super.setUp()
-
+        
         do {
+            connection = try! PostgreSQL.Connection(info: .init(URL(string: "postgres://localhost:5432/swift_test")!))
```

Comment:
Is there a reason the connection is being reconstructed each time? Is this the right way to use the api now?